### ISSUE TITLE
The q key should not exit helpSelectSearch

### ISSUE THREAD

**Issue by wfranzini** [NorfairKing/smos#43]
Hitting `q` in the help search line exits the help screen.

It should behave differently. since it's currently impossible to
search for a query related command, if any.

**Comment by NorfairKing** [comment]
I agree, this should be a one-line deletion.
I'll get to it.

**Comment by NorfairKing** [comment]
`c48d611` will fix this.

**Comment by wfranzini** [comment]
The master branch now has 2 tests failing for me:

```
... [lot of stuff]
To rerun use: --match "/Smos.ConfigFile/.json/default config/rebuilds to the the contents of /build/source/test_resources/config/json/complete.json/"

Randomized with seed 1217651565

Finished in 23.2323 seconds
303 examples, 2 failures
Test suite smos-test: FAIL
```

**Comment by NorfairKing** [comment]
@wfranzini Yes, I'm implementing #42 at the moment. I'll then fix the build and ping you on #42 so you can try both out.

Edit: I should have done that on `development` instead, but oh well.
